### ISSUE TITLE
SA2 - File to import not found or unreadable: compass/support

### ISSUE THREAD

**Issue by natework** [mhs/scout-app#239]
I am getting this error and have not been able to figure it out. If you need any more information please let me know. Thanks for the great tool!

OS: Windows 7 Enterprise (32-bit)

Information from Dev Tools: 

Error {formatted: "Error: File to import not found or unreadable: com…ss/_init.scss↵>> @import "compass/support";↵   ^↵", message: "File to import not found or unreadable: compass/su…t-App/Projects/revcyclesub - Copy/scss/_init.scss", column: 1, line: 172, file: "C:/Program Files/Scout-App/Projects/revcyclesub - Copy/scss/_init.scss"…}column: 1file: "C:/Program Files/Scout-App/Projects/revcyclesub - Copy/scss/_init.scss"formatted: "Error: File to import not found or unreadable: compass/support↵       Parent style sheet: C:/Program Files/Scout-App/Projects/revcyclesub - Copy/scss/_init.scss↵        on line 172 of Projects/revcyclesub - Copy/scss/_init.scss↵>> @import "compass/support";↵   ^↵"line: 172message: "File to import not found or unreadable: compass/support↵Parent style sheet: C:/Program Files/Scout-App/Projects/revcyclesub - Copy/scss/_init.scss"stack: (...)get stack: function () { [native code] }set stack: function () { [native code] }status: 1__proto__: d

(Attach screenshot below)
![scout-app 2 -issue screenshot](https://cloud.githubusercontent.com/assets/8547535/15720459/7e5b4920-2802-11e6-838a-0a9698c4b36c.JPG)

**Comment by TheJaredWilcurt** [comment]
Scout-App 2 uses Node-Sass instead of Ruby Sass. Which means it's much much faster and easier to keep up to date. Unfortunately that also means that Ruby-Specific aspects of the Sass ecosystem are not supported, such as Compass. We do include compass mixins out of the box, but that's only a portion of what Compass offers. And we only support the items in the root of the folder (`compass`, `animate`, and `lemonade`).

Since there is a `compass/support.scss` file that ships with the Compass-Mixins component, your best bet is to add it as a dependency in your `bower.json` file or just manually download the package from:

* https://github.com/Igosuki/compass-mixins

Then have your Sass files point directly to the mixin files, if you aren't just using the standard `_compass.scss` file.

**TLDR:** Scout-App 2 doesn't fully support Compass because it doesn't use Ruby. So you'll either need to rewrite your code so it doesn't depend on it, or manually set up your own Compass mixins.

**Comment by natework** [comment]
Thank you for the great explanation!

**Comment by claremansfield** [comment]
TheJaredWilcurt

THANK YOU SO MUCH! I've been trying to work out why Compass will not compile on macOS Mojave 10.14 all day!
